tests/unit/MiddlewareTest.ts

```diff
@@ -39,7 +39,7 @@ describe('Middleware', () => {
                 },
                 {
                     command: 'AddComment',
-                    data: {authToken: 'testToken', reportID: '1234', reportActionID: '5678', reportComment: 'foo'},
+                    data: {authToken: 'testToken', reportID: '1234', reportActionID: '5678'},
```

Comment:
Why are we removing the `reportComment` here? is that not needed at all in the test?

Reply:
It's not needed for the test.
This was wrong because Report doesn't have any reportComment, with the addition of the TS on this file the type started to complain.
This was working before as we are directly testing the Middleware and we expected exactly that data.
So in my opinion this can be safely removed from the test.

Reply:
Ok thank you!